RELEASING.rst

```diff
@@ -1,6 +1,10 @@
 Release Procedure
 -----------------
 
+#. Dependening on the magnitude of the changes in the release, consider testing
```

Comment:
Perhaps we should have a workflow which runs them all in parallel? This workflow can then be triggered manually and perhaps also in a cron and/or merge with `main`.

Reply:
Yes that would be good, but I am too lazy to do that ATM, but wanted to at least make it possible to test locally for now.